doc/text/news.md

```diff
@@ -1,5 +1,12 @@
 # News
 
+## 1.0.10 - 2019-12-08
+
+### Improvements
+
+  * Added Ubuntu support.
+    [Patch by İsmail Arılık](https://github.com/ruby-gnome/native-package-installer/pull/9)
```

Comment:
We use `[GitHub#N][Patch by XXX]` style in this file.

Reply:
ca617f7